src/main/webapp/WEB-INF/fenixedu-academic/registrationManagement/move-registrations.jsp

```diff
@@ -112,6 +112,33 @@ ${portal.toolkit()}
                 </c:forEach>
             </tbody>
         </table>
+        <h4><spring:message code="label.student.registrationManagement.phds" /></h4>
+        <table class="table">
+            <thead>
+                <tr>
+                    <th><spring:message code="label.student.registrationManagement.phd.year" /></th>
+                    <th><spring:message code="label.student.registrationManagement.phd.processNumber" /></th>
+                    <th><spring:message code="label.student.registrationManagement.phd.phdProgram" /></th>
+                    <th><spring:message code="label.student.registrationManagement.phd.state" /></th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                <c:forEach var="phd" items="${bean.target.person.phdIndividualProgramProcessesSet}">
+                    <tr>
+                        <td>${phd.executionYear.name}</td>
```

Comment:
Don't forget to add the `<c:out` tags, otherwise you may become vulnerable to XSS.